droidmediacodec.cpp

```diff
@@ -67,68 +67,70 @@
+    { android::MEDIA_MIMETYPE_VIDEO_AV1, android::kKeyAV1C, android::kKeyAV1C },
 #endif
 #if ANDROID_MAJOR >= 5
-    {android::MEDIA_MIMETYPE_VIDEO_HEVC, android::kKeyHVCC, android::kKeyHVCC},
+    { android::MEDIA_MIMETYPE_VIDEO_HEVC, android::kKeyHVCC, android::kKeyHVCC },
 #endif
-    {android::MEDIA_MIMETYPE_VIDEO_MPEG4, android::kKeyESDS, android::kTypeESDS},
-    {android::MEDIA_MIMETYPE_AUDIO_AAC, android::kKeyESDS, android::kTypeESDS},
-    {android::MEDIA_MIMETYPE_VIDEO_AVC, android::kKeyAVCC, android::kTypeAVCC},
-    {NULL, 0, 0}
+    { android::MEDIA_MIMETYPE_VIDEO_MPEG4, android::kKeyESDS, android::kTypeESDS },
+    { android::MEDIA_MIMETYPE_AUDIO_AAC, android::kKeyESDS, android::kTypeESDS },
+    { android::MEDIA_MIMETYPE_VIDEO_AVC, android::kKeyAVCC, android::kTypeAVCC }, { NULL, 0, 0 }
 };
 
 class Source : public android::MediaSource {
 public:
-    Source(android::sp<android::MetaData>& metaData) :
-        m_metaData(metaData),
-        m_running(false),
-        m_draining(false)
+    Source(android::sp<android::MetaData> &metaData)
+        : m_metaData(metaData), m_running(false), m_draining(false)
     {
     }
 
-    void unlock() {
+    void unlock()
+    {
         m_framesReceived.lock.lock();
         m_framesReceived.cond.broadcast();
         m_framesReceived.lock.unlock();
     }
 
-    // After this is called all new buffers will be rejected. get() will still return any already
-    // queued buffers but once the queue has been emptied it will return a null buffer indicating
-    // the end of the stream. There is no coming back from this.
-    void drain() {
+    // After this is called all new buffers will be rejected. get() will still
+    // return any already queued buffers but once the queue has been emptied it
+    // will return a null buffer indicating the end of the stream. There is no
+    // coming back from this.
+    void drain()
+    {
         m_framesReceived.lock.lock();
         m_draining = true;
         m_framesReceived.cond.broadcast();
         m_framesReceived.lock.unlock();
     }
 
-    void add(android::MediaBuffer *buffer) {
+    void add(android::MediaBuffer *buffer)
+    {
         m_framesReceived.lock.lock();
 
         // drain buffer gets added without checking to avoid a deadlock
 
         while (!m_draining) {
-        // TODO: I am not sure this is the right approach here.
+            // TODO: I am not sure this is the right approach here.
             if (m_framesReceived.buffers.size() < DROID_MEDIA_CODEC_MAX_INPUT_BUFFERS) {
                 m_framesReceived.buffers.push_back(buffer);
                 m_framesReceived.cond.signal();
                 m_framesReceived.lock.unlock();
 
                 return;
             }
-            // either get() will signal us or we will be signaled in case of an error
+            // either get() will signal us or we will be signaled in case of an
+            // error
```

Comment:
Looks like relatively short line length being forced here.

Reply:
Once we converged on the parameters to use I will rerun the formatting and create a fresh set of commits not containing this